soroban-spec/root/README.md

```diff
@@ -0,0 +1,55 @@
+# abundance-token JS
```

Comment:
Are these files supposed to be included?

Reply:
I don't know how they got here; no they are not